app/view_models/nsm/v1/additional_fee.rb

```diff
@@ -35,6 +35,16 @@ def t(key, width: nil, numeric: true, scope: 'nsm.additional_fees.index')
         end
       end
 
+      def backlink_path(claim)
+        # :nocov:
+        if any_adjustments?
+          Rails.application.routes.url_helpers.adjusted_nsm_claim_additional_fees_path(claim)
+        # :nocov:
+        else
+          Rails.application.routes.url_helpers.nsm_claim_additional_fees_path(claim)
+        end
+      end
+
```

Comment:
is the include_youth_court_fee attribute in this class still needed as it's captured in youth_court_fee as an adjustable attribute?

Reply:
removed attribute include_youth_court_fee as duplicated in child model, renamed additional_fee_spec to youth_court_fee_spec